uc-crux-llvm/src/UCCrux/LLVM/Main.hs

```diff
@@ -25,6 +25,7 @@ module UCCrux.LLVM.Main
     loopOnFunctions,
     translateLLVMModule,
     translateFile,
+    SomeModuleContext' (..),
```

Comment:
Because `SomeModuleContext` is already taken: https://github.com/GaloisInc/crucible/pull/708/files#diff-b78ee7aee50f1bf752da31af534521e16bc7a272aa4dbecbfa89205e9cd14e9aR125

It wasn't necessary before because `ModuleContext` only had one type parameter, but now that it has two there are four variants of which ones get existentially closed over...